docs-kits/kits/behaviour-twin-kit/development-view/overview.md

```diff
@@ -0,0 +1,42 @@
+﻿---
```

Comment:
Please also add the legal notice section here for the development view.

Reply:
Will be done.

Reply:
Done.